jira-create-issue/create_issue.js

```diff
@@ -0,0 +1,49 @@
+const core = require('@actions/core');
+const fetch = (...args) => import('node-fetch').then(({default: fetch}) => fetch(...args));
```

Comment:
We run in Node v20. The Fetch API is [natively supported in Node since v18](https://dev.to/andrewbaisden/the-nodejs-18-fetch-api-72m). Is there another reason that I am missing for why we are including an external library here?